src/RegExLib.Core/Entities/Author.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using RegExLib.Core.Events;
+using RegExLib.Core.Interfaces;
+using RegExLib.SharedKernel;
+
+namespace RegExLib.Core.Entities
+{
+    public class Author : BaseEntity
+    {
+        public string FullName { get; set; }
+        public string Username { get; set; }
+        public string UserId { get; set; }
+        public List<Expression> Expressions { get; private set; } = new List<Expression>();
+
+        public Author()
+        {
+        }
+
+        public Author AddExpression(Expression expression)
+        {
+            if (expression != null)
+            {
+                Expressions.Add(expression);
+            }
+
+            return this;
+        }
```

Comment:
What about adding multiple Expressions at one time? Should we accept `IEnumerable<Expression>` instead either here or in a different method?

Reply:
I added this 

```csharp 
        public Author AddExpression(IEnumerable<Expression> expressions)
        {
            if (expressions != null)
            {
                Expressions.AddRange(expressions);
            }

            return this;
        }
```

Reply:
AddExpressions (plural)

Reply:
BTW, this method may be a YAGNI violation as well. I was mostly asking the question as a discussion point. Perhaps @ardalis could weigh in here, too.

Reply:
Done

        public void AddExpressions(IEnumerable<Expression> expressions)
        {
            if (expressions != null)
            {
                Expressions.AddRange(expressions);
            }
        }
> 
> 
> AddExpressions (plural)

Reply:
My preference would be:
- AddExpression(expression) since there won't be any UI or user case for one author to add multiple at one time.
- Use Ardalis.GuardClauses for null checks but also
- Enable Nullable Reference Types so you probably don't even need the null check

Reply:
👍 NRTs has my vote. Then, guard clauses are only required on public WebAPI actions/endpoints until [parameter null checking](https://github.com/dotnet/csharplang/issues/2145) ships in C# 9.